frontend/src/components/Settings/ConfigurationForm.tsx

```diff
@@ -53,7 +53,7 @@ export const ConfigurationForm: FC<FormProps> = ({ formValues, onSubmit }) => {
                 inputName="grafanaUsername"
                 invalid={!!errors.grafanaUsername}
                 errorMessage={errors.grafanaUsername?.message ?? ''}
-                register={() => register({ required: intl.get('required') })}
+                register={() => register('username', { required: intl.get('required') })}
```

Comment:
For me, the plugin config page doesn't load without this.

Reply:
I suppose grafana is injecting in a newer version of grafana/ui ?

https://github.com/grafana/grafana/commit/3b515e650c55f0125020660d8aa22811f756b0d3#diff-f4a60751706bfa100bd13b2965a028d8e0e4db8a3592e0e701f26696afc3e5f1R267